### PR TITLE
Fixed rainbow dependency

### PR DESCRIPTION
**Pull request by lasseebert**
This should fix the dependency problem mentioned in #17

### CODE REVIEW

File: holepicker.gemspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.email = "jakub.suder@gmail.com"
 
   s.add_dependency 'json', '>= 1.7.7'
-  s.add_dependency 'rainbow', '>= 1.1.4'
+  s.add_dependency 'rainbow', '>= 1.1.4', '< 2.0.0'
```
Comment (mikegee) [inline holepicker.gemspec:16]: Here's the equivalent shorthand: `'~> 1.1'`

Comment (lasseebert) [inline holepicker.gemspec:16]: Wouldn't that also allow version `1.1.3`?

Comment (mikegee) [inline holepicker.gemspec:16]: Yes, but I'm having a hard time thinking of a scenario where that matters.

Comment (lasseebert) [inline holepicker.gemspec:16]: True. It proably doesn't matter. But IMHO the concern of this PR is to ensure that we don't use the incompatible version `2.0.0`. It's not to figure out weather `1.1.3` is usable.

There might be a historic reason for the earlier dependency constraint. I honor that (unknown) reason ;)

**Comment by mackuba** [comment]
I've fixed the problem the proper way ;) https://github.com/jsuder/holepicker/issues/17

**Comment by lasseebert** [comment]
Thanks, @jsuder :-)

**Pull request closed without merging**